### PR TITLE
JFrog cloud

### PR DESCRIPTION
**Pull request by danielmkn**
### What does this PR do?

Adding clarification to the readme file about the state of the integration. 

### Motivation

JFrog Cloud integration is currently in the Beta stage and won't be available to all customers yet.

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [x] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### CODE REVIEW

**Review by emarsha94 — APPROVED** [review]

File: jfrog_platform_cloud/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+JFrog Artifactory serves as your central hub for DevOps, integrating with your tools and processes to improve automation, increase integrity, and incorporate best practices along the way.
+
+JFrog's SaaS Log Streamer is a log streaming solution built by JFrog for SaaS customers. This solution will stream JFrog Artifactory logs from the customer's JFrog SaaS instance straight into their Datadog instance.
+
+Customers who use both JFrog and Datadog will be able to visualize Artifactory logs inside pre-configured Datadog dashboards. This integration also has built-in support for Datadog log pipelines which means logs streamed from JFrog will be preprocessed and automatically converted into the Datadog log format, allowing teams to uniquely name logs per their needs, drill down into Artifactory logs through searchable facets, and easily monitor their JFrog SaaS instance.
+
+At the initial release this integration will stream the following artifactory logs to Datadog:
```
Comment (alai97) [inline jfrog_platform_cloud/README.md:12]:
```suggestion
JFrog's SaaS Log Streamer is a log streaming solution built by JFrog for SaaS customers. This solution  streams JFrog Artifactory logs from the customer's JFrog SaaS instance into their Datadog instance.

Customers who use both JFrog and Datadog can visualize Artifactory logs with out-of-the-box Datadog dashboards. This integration also supports Datadog log pipelines, which means logs streamed from JFrog are preprocessed and automatically converted into the Datadog log format, allowing teams to uniquely name logs per their needs, drill down into Artifactory logs through searchable facets, and easily monitor their JFrog SaaS instance.

This integration enables you to stream the following artifactory logs to Datadog:
```

File: jfrog_platform_cloud/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- **access-audit.log**
+- **artifactory-request.log**
+- **access-security-audit.log**
+
+These logs will allow customers to readily know who accessed what repositories and how often. The logs will also show what IP addresses accessed those repositories. Log types such as traffic.log, artifactory-access.log and more request logs will be added to this integration in future updates.
+
+JFrog’s SaaS Log Streaming is currently in Beta. While in Beta, the cloud log streaming feature will only be available inside the MyJFrog portal for select JFrog Enterprise + customers. JFrog plans to GA this feature later in Q2 2024 at which point it will be available to all JFrog Enterprise + customers.
```
Comment (alai97) [inline jfrog_platform_cloud/README.md:20]: We don't typically talk about what features/product limitations will be delivered in future updates. Is it fine to leave this out and add this info back once this becomes available later on?

```suggestion
These logs allow you to readily know who accessed what repositories and how often. The logs also show what IP addresses accessed those repositories. 

JFrog’s SaaS Log Streaming is in beta. While in beta, the cloud log streaming feature will only be available inside the MyJFrog portal for select JFrog Enterprise + customers. 
```

File: jfrog_platform_cloud/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+## Setup
+
+**Note:** The integration requires JFrog Enterprise Plus subscription.
```
Comment (alai97) [inline jfrog_platform_cloud/README.md:24]:
```suggestion
This integration requires a JFrog Enterprise Plus subscription. 
```

File: jfrog_platform_cloud/README.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+JFrog’s SaaS Log Streaming is currently in Beta. While in Beta, the cloud log streaming feature will only be available inside the MyJFrog portal for select JFrog Enterprise + customers. JFrog plans to GA this feature later in Q2 2024 at which point it will be available to all JFrog Enterprise + customers.
+
+## Setup
+
+**Note:** The integration requires JFrog Enterprise Plus subscription. 
+
+### Installation
+
+Create [DataDog API key](https://app.datadoghq.com/organization-settings/api-keys).
```
Comment (alai97) [inline jfrog_platform_cloud/README.md:28]:
```suggestion
Create a [Datadog API key][2].  
```

File: jfrog_platform_cloud/README.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
+# Agent Check: JFrog Platform Cloud
+
+## Overview
+
+[JFrog](https://jfrog.com/) is a universal hybrid, end-to-end DevOps platform. JFrog Artifactory is the single solution for housing and managing all the artifacts, binaries, packages, files, containers, and components for use throughout your software supply chain.
```
Comment (alai97) [inline jfrog_platform_cloud/README.md:5]:
```suggestion
[JFrog][1] is a universal hybrid, end-to-end DevOps platform. JFrog Artifactory is the single solution for housing and managing all the artifacts, binaries, packages, files, containers, and components for use throughout your software supply chain.
```

File: jfrog_platform_cloud/README.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+**Note:** The integration requires JFrog Enterprise Plus subscription. 
+
+### Installation
+
+Create [DataDog API key](https://app.datadoghq.com/organization-settings/api-keys).  
+
+On the [MyJFrog Portal](https://my.jfrog.com), go to Settings -> JFrog Cloud Log Streaming - BETA, and enable the Log Streamer.
+Select Datadog as the vendor. Add your Datadog API key, the Datadog intake URL for your [Datadog site](https://docs.datadoghq.com/getting_started/site/) (`https://http-intake.logs.datadoghq.com/api/v2/logs`), and `ddtags`. Click Save.
+Your logs will start streaming into Datadog in 24 hours or less.
```
Comment (alai97) [inline jfrog_platform_cloud/README.md:32]:
```suggestion
On the [MyJFrog Portal][4], go to Settings -> JFrog Cloud Log Streaming - BETA, and enable the Log Streamer.

Select Datadog as the vendor. Add your Datadog API key, the Datadog intake URL for your [Datadog site][3] (for example: `https://http-intake.logs.datadoghq.com/api/v2/logs`), and `ddtags`. Click **Save**.

Your logs will start streaming into Datadog in 24 hours or less.
```

File: jfrog_platform_cloud/README.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+## Support
+
+Need help? Contact [JFrog Support](https://support.jfrog.com/).
```
Comment (alai97) [inline jfrog_platform_cloud/README.md:36]:
```suggestion
Need help? Contact [JFrog Support][5].

[1]: https://jfrog.com/
[2]: https://app.datadoghq.com/organization-settings/api-keys
[3]: https://docs.datadoghq.com/getting_started/site/
[4]: https://my.jfrog.com
[5]: https://support.jfrog.com/
```

**Review by alai97 — COMMENTED** [review]
Looks good, some suggestions!

**Comment by danielmkn** [comment]
Invalid PR

**Review by alai97 — COMMENTED** [review]
Looks good, some suggestions!

**Comment by danielmkn** [comment]
Sorry, @alai97, I've moved the actual change to a [new PR](https://github.com/DataDog/integrations-extras/pull/2227)

**Comment by alai97** [comment]
No worries, you can port over these suggestions in a follow up PR like https://github.com/DataDog/integrations-extras/pull/2227. Thanks!

**Pull request closed without merging**